tests/e2e/bulk-delete.js

```diff
@@ -56,35 +56,29 @@ describe('Bulk delete reports', () => {
   ];
 
   const savedUuids = [];
-  beforeEach(done => {
-    protractor.promise
-      .all(docs.map(utils.saveDoc))
-      .then(results => {
-        results.forEach(result => {
-          savedUuids.push(result.id);
-        });
-        done();
-      })
-      .catch(done.fail);
+  beforeEach(async () => {
+    const results = await Promise.all(docs.map(utils.saveDocNative));
+    results.forEach(result => {
+      savedUuids.push(result.id);
```

Comment:
When I first read this I was going to point out the awkward appending here instead of assignment. Then I realized it is on purpose (at least if the original code was "on purpose"). This shared state risks making tests dependent on each other having run (something we'd all talked about recently as wanting to get away from). I don't think today is the day to correct this, but this is a good one to remember for future improvements.

For now though, thanks for the async/await treatment here. This is much easier to read now!

Reply:
To clarify you are referring to the `savedUuids` being shared right?

Reply:
Yup.